examples/scale/frontend/frontend.go

```diff
@@ -89,13 +107,24 @@ func runner(fe pb.FrontendServiceClient) {
 	time.Sleep(time.Duration(rand.Int63n(int64(time.Second))))
 
 	g.start(mRunnersCreating)
+	createdAt := time.Now()
 	id, err := createTicket(ctx, fe)
 	if err != nil {
 		logger.WithError(err).Error("failed to create a ticket")
 		return
 	}
 
-	_ = id
+	err = watchAssignments(ctx, fe, ticketToWatch{id: id, createdAt: createdAt})
+	if err != nil {
+		logger.WithError(err).Errorf("failed to get ticket assignment: %s", id)
+	} else {
```

Comment:
i think this else block can be dropped and move the code to the top level, right??

Reply:
Can't agree with this because I measure ticket assignment time when call watchAssignments finishes successfully only.